src/ForeignController.sol

```diff
@@ -826,12 +826,11 @@ contract ForeignController is AccessControl {
     }
 
     function removeLiquidityUniswapV3(
-        address pool,
-        uint256 tokenId,
-        uint128 liquidity,
-        uint256 amount0Min,
-        uint256 amount1Min,
-        uint256 deadline
+        address                   pool,
+        uint256                   tokenId,
+        uint128                   liquidity,
+        UniswapV3Lib.TokenAmounts calldata min,
```

Comment:
variable names should like up like in `addLiquidityUniswapV3`

Reply:
https://github.com/grove-labs/grove-alm-controller/pull/67/commits/04c751043fded42dcb5833594997439f7dd17a85